recipes/empyre/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "empyre" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/ae/a6/057ff3e951365d86d83f63220d4f8893b7970df87f813426c791d2352839/empyre-0.3.2.tar.gz
+  sha256: 20ad0a16de0c2d3539c68b286a417721d3230da8f7789cd8804929fddc06df5d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - matplotlib-base >=3.2
+    - numpy >=1.17
+    - pillow
+    - python >=3.6
+    - scikit-image
+    - scipy
+    - tqdm
+
+test:
+  imports:
+    - empyre
+  commands:
+    - pip check
+    - pytest tests
+  source_files:
+    - tests
+  requires:
+    - pip
+    - pytest
+    - pytest
+    - pytest-cov
+    - pytest-flake8
+    - pytest-runner
+
+about:
+  home: https://iffgit.fz-juelich.de/empyre/empyre
+  summary: Electron Microscopy Python Reconstruction
+  license: GPL-3.0-or-later
+  license_file: LICENSE
+  doc_url: https://empyre.iffgit.fz-juelich.de/empyre/
+  dev_url: https://iffgit.fz-juelich.de/empyre/empyre
+
+extra:
+  recipe-maintainers:
+    - jan-car
+    - sk1p
+    - uellue
+    - matbryan52
```

Comment:
We need @jan-car, @sk1p, and @uellue to agree to become maintainers here.

Reply:
I agree!

Reply:
I agree, too. 👍

Reply:
I agree, as well 👍